{"uuid":"d67079db-6baf-45a7-b385-354702e616ee","name":"tests.LogoutTest","children":["d24c4b70-6665-4df1-b14a-c19f05e8bd35","6430a67f-fc5b-49f1-9f59-37b0db260d5f"],"befores":[{"name":"setup","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Appium HTTP server has started and is listening although we were unable to get an OK response from http://127.0.0.1:4723/wd/hub/status. Make sure both the client and the server use the same base path '/wd/hub/' and check the server log for possible error messages occurrences.\nResponse status code: 404\nResponse payload: {\"status\":9,\"value\":{\"error\":\"unknown command\",\"message\":\"The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource\",\"stacktrace\":\"\"}}\nNode.js executable path: /Users/md.abubakkarsiddik/.nvm/versions/node/v18.20.6/bin/node\nArguments: [/Users/md.abubakkarsiddik/.nvm/versions/node/v18.20.6/lib/node_modules/appium/build/lib/main.js, --port, 4723, --address, 0.0.0.0, --log, /Users/md.abubakkarsiddik/Documents/Findit-App-Automation/appium_server_logs.txt, --base-path, /wd/hub/, --relaxed-security]","trace":"io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server has started and is listening although we were unable to get an OK response from http://127.0.0.1:4723/wd/hub/status. Make sure both the client and the server use the same base path '/wd/hub/' and check the server log for possible error messages occurrences.\nResponse status code: 404\nResponse payload: {\"status\":9,\"value\":{\"error\":\"unknown command\",\"message\":\"The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource\",\"stacktrace\":\"\"}}\nNode.js executable path: /Users/md.abubakkarsiddik/.nvm/versions/node/v18.20.6/bin/node\nArguments: [/Users/md.abubakkarsiddik/.nvm/versions/node/v18.20.6/lib/node_modules/appium/build/lib/main.js, --port, 4723, --address, 0.0.0.0, --log, /Users/md.abubakkarsiddik/Documents/Findit-App-Automation/appium_server_logs.txt, --base-path, /wd/hub/, --relaxed-security]\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:192)\n\tat pages.BasePage.startAppiumServer(BasePage.java:312)\n\tat pages.BasePage.initializeDriver(BasePage.java:280)\n\tat tests.BaseTest.setup(BaseTest.java:94)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\n\tat org.testng.TestRunner.run(TestRunner.java:614)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\nCaused by: io.appium.java_client.service.local.AppiumServerAvailabilityChecker$ConnectionError: ConnectionError\n\tat io.appium.java_client.service.local.AppiumServerAvailabilityChecker.checkResponse(AppiumServerAvailabilityChecker.java:91)\n\tat io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:62)\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:144)\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:185)\n\t... 26 more\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1738497162892,"stop":1738497163140}],"afters":[{"name":"tearDown","status":"broken","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1738497163211,"stop":1738497163211}],"start":1738497162876,"stop":1738497163216}